src/commcare_cloud/environment/schemas/postgresql.py

```diff
@@ -294,6 +298,7 @@ class SmartDBConfig(jsonobject.JsonObject):
     ucr = jsonobject.ObjectProperty(lambda: UcrDBOptions, required=True)
     synclogs = jsonobject.ObjectProperty(lambda: SynclogsDBOptions, required=False)
     form_processing = jsonobject.ObjectProperty(lambda: FormProcessingConfig, required=False)
+    auditcare = jsonobject.ObjectProperty(lambda: AuditcareDBOptions, required=False)
```

Comment:
This change and/or the one above it appears to have caused an "auditcare" database to be created on every environment, which was not my intention. I only want an auditcare database if specified explicitly in `postgresql.yml` for the environment. I thought that's what `required=False` would do here, but I guess not? Is there some other option for that?

Reply:
Looks like you have to explicitly set it to `null` to have it excluded: https://github.com/dimagi/commcare-cloud/blob/master/.travis/environments/travis/postgresql.yml#L5.

I think if you made the default `None` it would be excluded unless specified in the yaml:

```
auditcare = jsonobject.ObjectProperty(lambda: AuditcareDBOptions, required=False, default=None)
```